### PR TITLE
feat: Add base governance addresses, CCTP, DaiUsds

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: src/Ethereum.sol [hunk 1]
```diff
@@ -173,9 +176,11 @@ library Ethereum {
     address internal constant SUPPLY_LOGIC      = 0x46256841e36b7557BB8e4c706beD38b17A9EB2c1;
 
     /******************************************************************************************************************/
-    /*** L2 Forwarders                                                                                              ***/
+    /*** Cross-Domain Addresses                                                                                     ***/
     /******************************************************************************************************************/
 
+    address internal constant CCTP_MESSENGER = 0xBd3fa81B58Ba92a82136038B25aDec7066af3155;
```
Comment (hexonaut) [inline src/Ethereum.sol:182]: This should probably be labelled `CCTP_TOKEN_MESSENGER` to be more clear.

File: src/Ethereum.sol [hunk 2]
```diff
@@ -7,29 +7,32 @@ library Ethereum {
     /*** Token Addresses                                                                                            ***/
     /******************************************************************************************************************/
 
+    address internal constant CBBTC  = 0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf;
```
Comment (hexonaut) [inline src/Ethereum.sol:10]: This was done in the last PR. Does master need to be merged in here?

File: src/Base.sol [hunk 3]
```diff
@@ -3,6 +3,13 @@ pragma solidity >=0.8.0;
 
 library Base {
 
+    /******************************************************************************************************************/
+    /*** Governance Relay Addresses                                                                                 ***/
+    /******************************************************************************************************************/
+
+    address internal constant SPARK_EXECUTOR = 0xF93B7122450A50AF3e5A76E1d546e95Ac1d0F579;
```
Comment (hexonaut) [inline src/Base.sol:10]: You need to revoke your deployer key as admin because the deploy script was incorrect.

**Review by hexonaut — CHANGES_REQUESTED** [review]

File: src/Ethereum.sol (hunk 1 above)
Comment (lucas-manuel) [reply src/Ethereum.sol]: Updated

File: src/Base.sol (hunk 3 above)
Comment (lucas-manuel) [reply src/Base.sol:10]: Revoked here https://basescan.org/tx/0xae30dfaaf43186f465e3c2b8d11a2d41fed798a7dce1437335d66a48f70bedf4

**Review by hexonaut — APPROVED** [review]

**Pull request merged**